vm/vm.go

```diff
@@ -378,6 +400,52 @@ func walkBinary(ctx expr.EvalContext, node *expr.BinaryNode) (value.Value, bool)
 				u.Errorf("at?%T  %v  coerce?%v bt? %T     %v", at, at.Value(), at.CanCoerce(stringRv), br, br)
 			}
 		}
+	case value.SliceValue:
+		switch node.Operator.T {
+		case lex.TokenContains:
+			switch br.(type) {
+			case nil, value.NilValue:
+				return nil, false
+			case value.StringValue:
+				// [x,y,z] contains str
+				for _, val := range at.Val() {
+					//u.Infof("str contains? %v %v", val.Value(), br.Value())
+					if eq, _ := value.Equal(val, br); eq {
```

Comment:
Oh I think contains is a useless operator. I'm just trying to match existing functionality.